modules/nf-core/pypgx/runngspipeline/tests/main.nf.test

```diff
@@ -0,0 +1,125 @@
+            }
+            run("PYPGX_PREPAREDEPTHOFCOVERAGE") {
+                script "../../preparedepthofcoverage"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'test'], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.PGx.CYP2D6.bam', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.PGx.CYP2D6.bam.bai', checkIfExists: true)
+                    ]
+                    input[1] = ["CYP2D6"] // Pharmacogene
+                    input[2] = "GRCh37" // assembly version
+                    """
+                }
+            }
+            run("PYPGX_COMPUTECONTROLSTATISTICS") {
+                script "../../computecontrolstatistics/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'test'], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.PGx.CYP2D6.bam', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.PGx.CYP2D6.bam.bai', checkIfExists: true)
+                    ]
+                    input[1] = "22:42512500-42551883"
+                    input[2] = "GRCh37" // assembly version
+                    """
+                }
+            }
+            run("UNTAR") {
+                script "../../../untar/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'resource_bundle'],
+                        file('https://github.com/sbslee/pypgx-bundle/archive/refs/tags/0.25.0.tar.gz')
+                    ]
+                    """
+                }
+            }
+        }
+        when {
+            process {
+                """
+                input[0] = PYPGX_CREATEINPUTVCF.out.vcf
+                            .join(PYPGX_CREATEINPUTVCF.out.tbi)
+                            .join(PYPGX_PREPAREDEPTHOFCOVERAGE.out.coverage)
+                            .join(PYPGX_COMPUTECONTROLSTATISTICS.out.control_stats)
+                            .combine(Channel.fromList(["CYP2D6"]))
+                input[1] = UNTAR.out.untar // resource bundle
+                input[2] = 'GRCh37' //assembly version
+                """
+            }
+        }
+        then {
+            def outDir = process.out.outdir[0][2]
+            def zipFile = path(outDir+"/results.zip").zip
+            def paths = zipFile.extractAll()
+            def metaFile = paths.find { it.getFileName().toString() == 'metadata.txt' }
+
+            assertAll(
+                { assert process.success },
+                { assert zipFile.isValid() },
+                { assert paths.size() == 2 },
+                { assert snapshot(
+                    metaFile,
```

Comment:
Is there only a `results.zip` inside the output ?
Could you snapshot all the files names present inside the zip file ?